mdl/src/test/java/org/tsi/mdlt/test/bdsql/BdsqlSyncTest.java

```diff
@@ -125,33 +132,39 @@ public void existingNamespaceNewObjectTest() throws IOException, InterruptedExce
         LogVerification("Existing ldap user with schema permission can access new table");
         String jdbcUrl = getValidPrestoJdbcUrl(schema);
         String selectQuery = getSelectQuery(schema, objectName);
-        assertEquals(2, executePrestoSelect(selectQuery, jdbcUrl, User.getLdapMdlAppUser()).size());
+        assertEquals(2, executePrestoSelect(selectQuery, jdbcUrl, User.getLdapSecAppUser()).size());
 
         LogVerification("Existing ldap user without schema permission cannot access new table");
         SQLException authorizationException = assertThrows(SQLException.class, () -> {
-            executePrestoSelect(selectQuery, jdbcUrl, User.getLdapSecAppUser());
+            executePrestoSelect(selectQuery, jdbcUrl, User.getLdapMdlAppUser());
         });
         verifyErrorMsgIsAccessDenied(authorizationException);
+
+        LogStep("Clean up test data");
+        cleanupBusinessData(HERD_ADMIN_USER, getBusinessObjectData(schema, objectName), false);
     }
 
     @Test
     /**
-     * verify existing ldap user access to new herd object with new namespace, existing ldap user authorization works correctly
+     * verify existing ldap user access to new herd object with new public namespace, existing ldap user authorization works correctly
      */
-    public void newNamespaceNewObjectTest() throws IOException, InterruptedException, ClassNotFoundException, NamingException, SQLException {
+    public void newPublicNamespaceNewObjectTest() throws IOException, InterruptedException, ClassNotFoundException, NamingException, SQLException {
         String schema = "mdlt_demo_data";
         String objectName = "testdata";
 
-        LogStep("Add Ldap user without AD group and sync");
-        String username = LDAP_USER_NAMESPACE;
+        String username = LDAP_NEWPUBLICNAMESPACE_NEWUSER;
         String password = "ldapPwd";
         User ldapUser = new User(username, password);
+
+        LogStep("Cleanup before test");
+        deleteEntryIgnoringError(username);
+
+        LogStep("Add Ldap user without AD group and sync");
         LdapUtil.addEntry(ldapUser);
-        syncBdsqlAuth();
         TimeUnit.SECONDS.sleep(15);
 
         LogStep("Create new namespace, herd object formats, object definition and upload herd data");
-        executeShellScript(PRESTO_NEW_OBJECT_SH, schema, objectName, "true");
+        executeShellScript(PRESTO_NEW_OBJECT_SH, schema, objectName, Boolean.toString(true));
```

Comment:
Is this different from what you had before?

Reply:
just remove the hardcode string